src/nuc/selector.py

```diff
@@ -64,16 +96,34 @@ def apply(self, value: Dict[str, Any]) -> Any:
         value
             The dict that this selector should be applied to.
         """
+        match self.target:
+            case SelectorTarget.TOKEN:
+                return self._apply_on_value(self.path, value)
+            case SelectorTarget.CONTEXT:
+                if not self.path:
+                    return None
+                first = self.path[0]
+                context_value = context.get(first)
+                if not context_value:
```

Comment:
Did you mean `if context_value is None`, because values like `0`, `False` and `""` would eval to `False`